uint32 frame_number
float64 t_capture
float64 t_sent
uint32 camera_id
DetectionBall[] balls
DetectionRobot[] robots_yellow
DetectionRobot[] robots_blue
